packages/fx-core/tests/core/middleware/projectMigrationV3.test.ts

```diff
@@ -395,6 +430,10 @@ async function readSettingJson(projectPath: string): Promise<any> {
   return await fs.readJson(path.join(projectPath, Constants.settingsFilePath));
 }
 
+async function readEnvFile(projectPath: string, env: string): Promise<any> {
+  return await fs.readFileSync(path.join(projectPath, ".env." + env)).toString();
```

Comment:
.env file should under `teamsfx` folder?

Reply:
maybe the same with the issue bellow